pkg/prowgen/prowgen_images_discovery.go

```diff
@@ -36,16 +37,23 @@ var (
 		"openshift/ci-operator/knative-images.*",
 		"openshift/ci-operator/knative-test-images.*",
 		"openshift/ci-operator/static-images.*",
+		"openshift/ci-operator/.*images?.*",
+	}
+	defaultDockerfileExcludes = []string{
+		"openshift/ci-operator/source-image.*",
+		"openshift/ci-operator/build-image.*",
 	}
 )
 
-type orgRepoTag struct {
-	Org  string
-	Repo string
-	Tag  string
+type ociReference struct {
+	Domain string
+	Org    string
+	Repo   string
+	Tag    string
+	Digest string
```

Comment:
Why is digest needed here?

Reply:
We don't use it currently, which doesn't mean at any point such image reference might be used in one of Dockerfiles.